test/onnx/test_utility_funs.py

```diff
@@ -13,6 +13,8 @@
 
 
 class TestUtilityFuns(TestCase):
+    from torch.onnx.symbolic_helper import _export_onnx_opset_version
+    opset_version = _export_onnx_opset_version
```

Comment:
do we just want explicit use opset 9? since we already cache opset_version here, and it won't change in the concrete test cases.

Reply:
Good point. Using explicit opset 9 here.